src/Nakama/IHttpAdapterExt.cs

```diff
@@ -0,0 +1,28 @@
+
+using System.Collections.Generic;
+
+namespace Nakama
+{
+    /// <summary>
+    /// Extension methods for the <see cref="IHttpAdapter"> interface.
+    /// </summary>
+    public static class IHttpAdapterExt
```

Comment:
Do we need the code as a separate method or extension method if its only called in one place at the moment? Will it need to be called from the Js HTTP adapter in the Unity client code?

Reply:
@novabyte it will need to be called by the `UnityWebRequestAdapter`

Reply:
Ah yes. Good point.